script.js

```diff
@@ -115,16 +115,18 @@
 
     pretty(options) {
       options = options || {};
-      var opts = Object.assign({}, options, {
+      var opts = Object.assign({
```

Comment:
Assigning options, then defaults was allowing defaults to override options, so reverse them.